editor/editor_help.cpp

```diff
@@ -2384,6 +2384,7 @@ static void _add_text_to_rt(const String &p_bbcode, RichTextLabel *p_rt, Control
 	List<String> tag_stack;
 	bool code_tag = false;
 	bool codeblock_tag = false;
+	const int code_indent_type = EDITOR_GET("text_editor/behavior/indent/type");
```

Comment:
Changing this setting won't update currently opened help pages.
Not sure if it's worth handling though.